<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditComputer</class>
 <widget class="QDialog" name="EditComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="editComputerLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Edit computer&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="editComputerNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editComputerNameInput"/>
   </item>
   <item>
    <widget class="QLabel" name="editComputerTypeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="editComputerType">
     <item>
      <property name="text">
       <string>Electronic</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mechanical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Transistor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="editComputerWasBuiltLabel">
     <property name="text">
      <string>Was it built?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="editComputerWasBuilt">
     <item>
      <property name="text">
       <string>Yes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unknown</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="editComptuerWasBuiltLabel">
     <property name="text">
      <string>Year built (optional):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="editComputerYearBuilt"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonEditComputer">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancelEditComputer">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="editComputerFeedbackLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
